velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -72,4 +73,78 @@ TEST_F(PlanBuilderTest, invalidAggregateFunctionCall) {
           .planNode(),
       "Aggregate function doesn't exist: maxx.");
 }
+
+namespace {
+
+void registerWindowFunction() {
+  std::vector<exec::FunctionSignaturePtr> signatures{
+      exec::FunctionSignatureBuilder()
+          .argumentType("BIGINT")
+          .returnType("BIGINT")
+          .build(),
+  };
+  exec::registerWindowFunction("window1", std::move(signatures), nullptr);
+}
+} // namespace
+
+TEST_F(PlanBuilderTest, windowFunctionCall) {
+  VELOX_ASSERT_THROW(
+      PlanBuilder()
+          .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+          .window({"window1(c) over (partition by a order by b) as d"})
+          .planNode(),
+      "Registry of window functions is empty.");
+
+  registerWindowFunction();
+
+  VELOX_CHECK_NOT_NULL(
```

Comment:
It would be nice to implement toString() and use it here to ensure no information is long in translation.

Reply:
I have toString() implemented in my next PR. Will leave a note here to change this test post that change if thats okay.